diff --git a/node_modules/@reach/router/es/index.js b/node_modules/@reach/router/es/index.js
index a20d8dc..e976b8c 100644
--- a/node_modules/@reach/router/es/index.js
+++ b/node_modules/@reach/router/es/index.js
@@ -616,4 +616,4 @@ var shouldNavigate = function shouldNavigate(event) {
 };
 
 ////////////////////////////////////////////////////////////////////////
-export { Link, Location, LocationProvider, Match, Redirect, Router, ServerLocation, createHistory, createMemorySource, isRedirect, navigate, redirectTo, globalHistory };
\ No newline at end of file
+export { Link, Location, LocationContext, LocationProvider, Match, Redirect, Router, ServerLocation, createHistory, createMemorySource, isRedirect, navigate, redirectTo, globalHistory };
